code/optimize/src/test/java/com/adobe/marketing/mobile/optimize/OptimizeExtensionTests.java

```diff
@@ -2152,4 +2153,15 @@ private void setConfigurationSharedState(
                                 ArgumentMatchers.eq(SharedStateResolution.ANY)))
                 .thenReturn(new SharedStateResult(status, data));
     }
+
+    @Test
+    public void testUpdateRequestEventIdsInProgressIsConcurrentHashMap() {
```

Comment:
Can you remove these tests? They are just implementation details in the Optimize extension and aren’t needed.

Reply:
done @praveek